### PR TITLE
feat(clerk-js,backend,types): Add experimental image_url field on users/orgs

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [x] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [x] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### CODE REVIEW

File: packages/types/src/signIn.ts [hunk 1]
```diff
@@ -109,6 +109,10 @@ export interface UserData {
   firstName?: string;
   lastName?: string;
   profileImageUrl?: string;
+  /**
+   * @experimental
+   */
+  imageUrl: string;
```
Comment (dimkl) [inline packages/types/src/signIn.ts:115]: 🔧 make `imageUrl?: string` optional to resolve the build issue.

**Review by dimkl — APPROVED** [review]

File: packages/types/src/json.ts [hunk 2]
```diff
@@ -297,6 +309,10 @@ export interface UserDataJSON {
   first_name?: string;
   last_name?: string;
   profile_image_url?: string;
+  /**
+   * @experimental
+   */
+  image_url: string;
```
Comment (dimkl) [inline packages/types/src/json.ts:315]: 🙃 Probably this needs to be `image_url?`

**Review by SokratisVidros — APPROVED** [review]

File: packages/clerk-js/src/core/resources/Organization.ts [hunk 3]
```diff
@@ -25,6 +25,7 @@ export class Organization extends BaseResource implements OrganizationResource {
   name!: string;
   slug!: string;
   logoUrl!: string;
+  experimental_imageUrl!: string;
```
Comment (dimkl) [inline packages/clerk-js/src/core/resources/Organization.ts:28]: ❓ why did we use `experimental_imageUrl` instead of `imageUrl` ?

**Review by dimkl — APPROVED** [review]

**Review by SokratisVidros — APPROVED** [review]

File: packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx [hunk 4]
```diff
@@ -36,8 +35,9 @@ const createFakeMember = (params: FakeMemberParams): OrganizationMembershipResou
       firstName: params?.firstName || 'test_firstName',
       lastName: params?.lastName || 'test_lastName',
       profileImageUrl: params?.profileImageUrl || '',
+      experimental_imageUrl: params?.experimental_imageUrl || '',
     },
-  };
+  } as any;
```
Comment (dimkl) [inline packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx:40]: ❓ is `as any` required ?

File: packages/clerk-js/src/utils/user.ts [hunk 5]
```diff
@@ -33,5 +35,12 @@ export const getIdentifier = (user: Partial<UserResource>): string => {
...
 export const isDefaultProfileImage = (url: string) => {
-  return !(url || '').includes('gravatar');
+  let encoded: string;
+  try {
+    encoded = new URL(url).pathname.replace('/', '');
+  } catch {
+    return false;
+  }
+  const decoded = decodeBase16(encoded);
+  return decoded.includes('default');
```
Comment (dimkl) [inline packages/clerk-js/src/utils/user.ts:45]: 🔧  i think this function should be with both `profileImageUrl` and `imageUrl`!

**Review by dimkl — APPROVED** [review]

File: packages/clerk-js/src/utils/user.ts (hunk 5 above)
Comment (nikosdouvlis) [reply packages/clerk-js/src/utils/user.ts:45]: Correct, fixing

Comment (nikosdouvlis) [reply packages/clerk-js/src/utils/user.ts]: Done, thanks @dimkl

**Pull request merged**